### PR TITLE
Feat/delegate view 228

### LINKED ISSUE #219: Select Position
we are going to put the search bar by name more in the center of the screen so that the user has a better visualization of it

#228 

Image

### LINKED ISSUE #228: Delegate View
Update the delegate view based on design https://www.figma.com/file/ylfjuHS0n5gHg4OgUUZJV9/Spend-Explorer

Image

### PR DESCRIPTION
**Pull request by JAMares**
### What does this PR do?

- Resolve #228 
- Resolve #219 

### Steps to test

1. Run project 
2. Enter to Delegate and should see this
imagen

#### CheckList

- [X] Follow proper Markdown format
- [X] The content is adequate
- [X] The content is available in both english and spanish
- [X] I Ran a spell check

### CODE REVIEW

File: webapp/src/components/StackedBarChartReport/index.js [hunk 1]
```diff
@@ -0,0 +1,250 @@
...
+        <Box
+          width={12}
+          height={12}
+          ml={2}
+          mt={0.5}
+          mr={0.5}
+          bgcolor="#19647e"
+          borderRadius={5}
+        />
+        Claimed
```
Comment (AngeloCG97) [inline webapp/src/components/StackedBarChartReport/index.js:56]: Add this text on translations jsons

File: webapp/src/components/StackedBarChartReport/index.js [hunk 2]
```diff
@@ -0,0 +1,250 @@
...
+        <Box
+          width={12}
+          height={12}
+          ml={2}
+          mt={0.5}
+          mr={0.5}
+          bgcolor="#ee964b"
+          borderRadius={5}
+        />
+        Categorized
```
Comment (AngeloCG97) [inline webapp/src/components/StackedBarChartReport/index.js:68]: Add this text on translations jsons

File: webapp/src/components/StackedBarChartReport/index.js [hunk 3]
```diff
@@ -0,0 +1,250 @@
...
+        <Box
+          width={12}
+          height={12}
+          ml={2}
+          mt={0.5}
+          mr={0.5}
+          bgcolor="#f4d35e"
+          borderRadius={5}
+        />
+        Unclaimed
```
Comment (AngeloCG97) [inline webapp/src/components/StackedBarChartReport/index.js:80]: Add this text on translations jsons

File: webapp/src/components/StackedBarChartReport/index.js [hunk 4]
```diff
@@ -0,0 +1,250 @@
...
+        <Box
+          width={12}
+          height={12}
+          ml={2}
+          mt={0.5}
+          mr={0.5}
+          bgcolor="#28afb0"
+          borderRadius={5}
+        />
+        Uncategorized
```
Comment (AngeloCG97) [inline webapp/src/components/StackedBarChartReport/index.js:92]: Add this text on translations jsons

**Review by AngeloCG97 — CHANGES_REQUESTED** [review]

File: webapp/src/routes/DelegateReport/index.js [hunk 5]
```diff
@@ -16,61 +21,161 @@ const useStyles = makeStyles(styles)
...
+  }, [delegateList, maxLevel])
+
+  useEffect(() => {
+    setAccordionList(
+      delegateList.filter(delegate => delegate?.account?.includes(searchValue))
+    )
+  }, [searchValue, delegateList])
+
+  accordionList.sort((d1, d2) =>
+    d1?.totalRewarded < d2?.totalRewarded
```
Comment (tetogomez) [inline webapp/src/routes/DelegateReport/index.js:105]: avoid nested ternary if

File: webapp/src/routes/DelegateReport/index.js [hunk 6]
```diff
@@ -16,61 +21,161 @@ const useStyles = makeStyles(styles)
...
+      ? 1
+      : d1?.totalRewarded > d2?.totalRewarded
+      ? -1
+      : 0
+  )
+
+  accordionList.sort((d1, d2) =>
+    d1?.rank?.memberType < d2?.rank?.memberType
```
Comment (tetogomez) [inline webapp/src/routes/DelegateReport/index.js:113]: avoid nested ternary if

**Review by tetogomez — CHANGES_REQUESTED** [review]

**Review by AngeloCG97 — APPROVED** [review]
LGTM

**Review by tetogomez — APPROVED** [review]
LGTM

**Pull request merged**